engine/src/main/resources/assets/i18n/menu_en.lang

```diff
@@ -218,6 +224,10 @@
     "ssao": "SSAO",
     "start-game": "Play",
     "start-singleplayer-game": "Singleplayer",
+    "telemetry-button": "Metric Menu",
```

Comment:
Like the other spot earlier this might be better as "Metrics Menu" (in other places something like "a metric" works as it clearly appears to be a regular noun)